test/images/webhook/Makefile

```diff
@@ -14,6 +14,7 @@
 
 build:
 	CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o webhook .
-	docker build --no-cache -t gcr.io/kubernetes-e2e-test-images/k8s-sample-admission-webhook-amd64:1.8v1 .
+	docker build --no-cache -t gcr.io/kubernetes-e2e-test-images/k8s-sample-admission-webhook-amd64:1.8v2 .
+	rm -rf webhook
 push:
-	gcloud docker --push gcr.io/kubernetes-e2e-test-images/k8s-sample-admission-webhook-amd64:1.8v1 .
+	gcloud docker -- push gcr.io/kubernetes-e2e-test-images/k8s-sample-admission-webhook-amd64:1.8v2
```

Comment:
Is the added space in `-- push` intentional?

Reply:
Yes, the original code doesn't work. Awkward :(

Reply:
Does just plain `gcloud docker push` not work?

Reply:
No. It seems that changed the api: https://cloud.google.com/container-registry/docs/pushing-and-pulling

Reply:
ack.